src/sympc/protocol/spdz/spdz.py

```diff
@@ -52,10 +54,11 @@ def mul_master(
         raise ValueError(f"{op_str} should be in {EXPECTED_OPS}")
 
     session = x.session
-
     shape_x = tuple(x.shape)
     shape_y = tuple(y.shape)
 
+    args = [list(el) + [op_str] for el in zip(x.share_ptrs, y.share_ptrs)]
+
     args = [
         list(el) + [op_str]
         for el in zip(session.session_ptrs, x.share_ptrs, y.share_ptrs)
```

Comment:
The same `args` variable is assigned twice here.

Reply:
it should be removed